code/game/mecha/combat/marauder.dm

```diff
@@ -41,6 +41,7 @@
 	ME = new /obj/item/mecha_parts/mecha_equipment/antiproj_armor_booster(src)
 	ME.attach(src)
 	max_ammo()
+	unlock_mech_weapons() //Mech Weapon Firing Pins PR by Surrealistik Oct 2019

```

Comment:
Is the intent here to unlock weapon restrictions for non-station combat mechs, or just for syndicate mechs?

If it's the former you've missed a couple.